cobra/test/test_core/test_summary/test_metabolite_summary.py

```diff
@@ -105,4 +106,7 @@ def test_metabolite_summary_to_frame_with_fva(model, opt_solver, fraction,
 
     out_df = model.metabolites.get_by_id(met).summary(fva=fraction).to_frame()
 
-    assert out_df['PERCENT'].tolist() == expected_percent
+    assert np.allclose(
```

Comment:
You could also use [pytest.approx](https://docs.pytest.org/en/latest/reference.html#pytest-approx). Shouldn't make a difference, though.

Reply:
Cool, didn't know that one :smile: Just stuck with what the other tests use...